pkgs/games/shticker_book_unwritten/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, rustPlatform, openssl, pkg-config, steam-run-native, lib }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "shticker_book_unwritten";
```

Comment:
If the name is adjusted in `pkgs/top-level/all-packages.nix`, then it should also be adjusted here, and the package path itself should be changed, too.